src/database/__tests__/firestore.spec.tsx

```diff
@@ -0,0 +1,189 @@
+import { of } from 'rxjs';
```

Comment:
I'm doing a lot of black magic with observables in this file so lmk if you have any questions

Reply:
I thought black magic was prohibited by the church but ok.